core/lib/generators/solidus/install/app_templates/frontend/starter.rb

```diff
@@ -1 +1 @@
-apply 'https://github.com/solidusio/solidus_starter_frontend/raw/main/template.rb'
+apply 'https://github.com/solidusio/solidus_starter_frontend/raw/bump-solidus_support/template.rb'
```

Comment:
Needs to be reverted to main after https://github.com/solidusio/solidus_starter_frontend/pull/394 got merged

Reply:
Do you mind reverting this so we can merge? I think this has [fixed my PR](https://github.com/solidusio/solidus/pull/6036)! (I lifted your changes in a wip commit and ran CI and it passed!)

Reply:
Done